v2/cmd/controller/main.go

```diff
@@ -169,6 +159,33 @@ func main() {
 	}
 }
 
+func getGetDefaultCredential(cfg config.Values, setupLog logr.Logger) (azcore.TokenCredential, error) {
+
+	// If subscriptionID is not supplied, then set default credential to not be used/nil
+	if cfg.SubscriptionID == "" {
+		setupLog.Info("No default credential set found, continuing without default credential")
```

Comment:
Are we calling this a _global_ credential or a _default_ one? 

Our users will be reading both the documentation and the logs - we should be consistent, else we run the risk of being confusing.

Remember also that some associate `default` with failure - when you _default_ on a loan, you've failed to pay it back.

Also, is this one credential, or many? A "credential set" implies one or more different credentials, but that doesn't match the code.

Maybe this?

```suggestion
		setupLog.Info("No global credential configured, continuing without.")
```